packages/js/product-editor/changelog/add-39893

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Show in product details feature under Edit attribute modal
```

Comment:
This description is not so descriptive, here's a suggestion:
```suggestion
Add 'Show in product details' checkbox under Edit attribute modal
```

Reply:
done -> https://github.com/woocommerce/woocommerce/pull/40006/commits/4d5c102411803743aab8562caef9056ae9eb0957